doc/api/debugger.md

```diff
@@ -197,8 +197,8 @@ V8 Inspector can be enabled by passing the `--inspect` flag when starting a
 Node.js application. It is also possible to supply a custom port with that flag,
 e.g. `--inspect=9222` will accept DevTools connections on port 9222.
 
-To break on the first line of the application code, provide the `--debug-brk`
-flag in addition to `--inspect`.
+To break on the first line of the application code, pass the `--inspect-brk`
```

Comment:
I'm not going to stop or even slow this PR from landing over this incredibly minor thing I'm about to bring up, but I gotta ask: Why the `provide` -> `pass` change? (I think of `pass` as something you do with arguments, not command-line flags, and so `provide` feels more correct to me in this context. But I'm not sure if that's my own idiosyncrasy.)